cgeglio/Overlook
# Show manager details

#### What's this PR do?
- Adds data to popout windows for the manager and customer dashboards
- Change around where certain functions live => moved several functions that populate user data from main js to the user class
- Adds images for pop up windows
#### Where should the reviewer start?
This one i kind of all over the place: most files were used. I would suggest looking in the main js first and following the method trail through different classes.
#### How should this be manually tested?
If you login as a manager or customer and click on any dashboard button, you should see a popup window that displays data pertaining to the dashboard section.
#### Any background context you want to provide?
@grayson-palmer right now I'm using a bunch of super similar functions in index.js and just changing the words manager and customer. I tried to DRY it up with interpolation but was running into trouble/errors. If you have any ideas - lemme know!
#### What are the relevant tickets?
#31 
@hannahhch
@khalidwilliams
